cocos/tween/actions/action-instant.ts

```diff
@@ -70,8 +70,8 @@ export class ActionInstant extends FiniteTimeAction {
  * @extends ActionInstant
  */
 export class Show extends ActionInstant {
-    update (dt: any): void {
-        const _renderComps = this.target!.getComponentsInChildren(Renderer);
+    update (_dt: number): void {
+        const _renderComps = (this.target as any).getComponentsInChildren(Renderer);
```

Comment:
Show and Hide actions could only be applied to `Node` target, and we don't want to make `tween` module depends `scene-graph`, so here we convert it to any type to access `getComponentsInChildren` method.

Reply:
Can change to `as Node`? As it should be Node here.

Reply:
I will check that.

Reply:
Updated.